package.json

```diff
@@ -1,37 +1,37 @@
 {
 	"name": "block-editor-governance",
-	"version": "1.0.9",
+	"version": "1.0.10",
 	"description": "This is a plugin adding additional governance capabilities to the block editor.",
 	"author": "VIP Bistro",
 	"main": "build/index.js",
 	"devDependencies": {
-		"@automattic/eslint-plugin-wpvip": "^0.11.0",
+		"@automattic/eslint-plugin-wpvip": "^0.13.0",
 		"@playwright/test": "^1.44.1",
-		"@wordpress/block-editor": "^13.0.0",
+		"@wordpress/block-editor": "^14.0.0",
 		"@wordpress/components": "^28.0.0",
-		"@wordpress/compose": "^7.0.0",
-		"@wordpress/data": "^10.0.0",
-		"@wordpress/e2e-test-utils-playwright": "^1.0.0",
-		"@wordpress/hooks": "^4.0.0",
+		"@wordpress/compose": "^7.2.0",
+		"@wordpress/data": "^10.2.0",
+		"@wordpress/e2e-test-utils-playwright": "^1.7.0",
+		"@wordpress/hooks": "^4.7.0",
 		"@wordpress/i18n": "^5.0.0",
-		"@wordpress/notices": "^5.0.0",
-		"@wordpress/scripts": "^27.0.0",
+		"@wordpress/notices": "^5.5.0",
+		"@wordpress/scripts": "^27.9.0",
 		"babel-jest": "^29.7.0",
 		"eslint": "^8.56.0",
 		"husky": "^9.0.11",
 		"lint-staged": "^15.2.5",
 		"phplint": "^2.0.5",
-		"prettier": "npm:wp-prettier@2.8.5"
+		"prettier": "npm:wp-prettier@2.8.5",
+		"webpack": "^5.91.0"
 	},
 	"scripts": {
+		"build": "webpack --mode production",
+    	"dev": "webpack --watch",
```

Comment:
Minor, there's an tab/space combo on this line.

Reply:
This might be a prettier bug clashing with my markdown settings. Lemme fix that